### PR TITLE
All quiz page

### PR DESCRIPTION
**Pull request by zoef-yt**
In this pr, I added a quiz page where all the quizzes by categories will be shown.

### Files to check
All files

Preview link

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,8 +4,114 @@
...
+				<a href="/"> Quiz-E-Games </a>
+			</div>
+			<div class="navbar-cta">
+				<img class="theme-changer theme-changer-icon" src="/assets/svg/sun.svg" alt="dark theme" />
+
+				<a id="login-text" class="btn btn-link">Hello, Sign Up</a>
+
+				<!-- <img
+					class="avatar avatar-sm"
+					src="https://media-exp1.licdn.com/dms/image/C4E03AQEm0ZBjaIr1hg/profile-displayphoto-shrink_200_200/0/1610261504692?e=1649894400&amp;v=beta&amp;t=kKO77EwqflEDsrZf5eG7xC7ZUB4hD_BkZuRJyFiFHbI"
```
Comment (Shubhamkr2610) [inline index.html]: suggestion : remove unused code

**Review by Shubhamkr2610 — APPROVED** [review]
suggested , Rest everything is good .
appreciation: added dark mode as extra feature is great and your code is good , clean , readable

**Review by dev-prabhat — APPROVED** [review]
Great Work on UI, Code structure is good, appreciate the class name of CSS

File: index.html (hunk 1 above)
Comment (zoef-yt) [reply index.html]: Will do that.

**Pull request merged**